lib/Jsrt/ChakraDebug.h

```diff
@@ -22,6 +22,14 @@
 
 #include "ChakraCommon.h"
 
+#if !defined(_WIN32) || !defined(_MSC_VER)
```

Comment:
Copied this from "ChakraCommon.h" -- non-Win32, or non-VS (clang?) on Win32. Better than __clang__?